docs/howto/query.md

```diff
@@ -22,7 +22,7 @@ type Location struct {
 
 > See more: {ref}`types` 
 
-## 2. Write your query string using the types
+## Write your query string using the types
 
 SQLair queries are regular SQL queries with special input and output expressions
 which allow you to reference the Go types directly from the SQL.
```

Comment:
```suggestion
which allow you to reference the Go types directly from SQL.
```

Reply:
Mean to say, I felt the numbers were a bit out of place in this article cuz we don't use them anywhere else in the how-tos plus making how to query a 5 step process sounds like quite a lot for a "convenience" library